docs/user/hbase/install.rst

```diff
@@ -171,6 +197,10 @@ A few suggested configurations are below:
             bin/install-hadoop.sh lib
             bin/install-hbase.sh lib
 
+        .. note::
+
+          You should adjust the versions at the top of each ``install-`` script to match your installation.
```

Comment:
I wonder if we should create 'install-chd-hadoop.sh' scripts that have the versions sorted?  (Where CDH be any distro/vendor specific bit.)

Reply:
I added that as an idea on https://geomesa.atlassian.net/browse/GEOMESA-2383